services/user/CommonSys/CMakeLists.txt

```diff
@@ -4,15 +4,4 @@ endfunction(add)
 
 include(ExternalProject)
 
-downloadThirdParty("htm" "https://unpkg.com/htm@3.1.0/dist/htm.module.js")
```

Comment:
The build is failing because the root CMake is still trying to copy these.

Reply:
Actually I'll do this cleanup in a different pr